src/applications/personalization/profile360/reducers/index.js

```diff
@@ -94,11 +96,18 @@ function vaProfile(state = initialState, action) {
         state,
       );
 
+    case PAYMENT_INFORMATION_FETCH_STARTED:
```

Comment:
I think having the three actions for the fetch is an improvement to what was happening before, but it seems like the changes in this reducer change the behavior of the page and I'm not sure that's correct?

Reply:
I'm gonna double-check everything with the fetch & page-behavior.  I might've been over-ambitious adding the actions.

Might push more changes up for this.

Reply:
Separated fetch-started reducer, making no state-changes now -- isLoading prop is controlled by other props.

Yep, piggy-backing the fetch-started action onto save-started was flipping the isSaving flag and impact the edit-modal behavior.